.buildscript/prepare_mkdocs.sh

```diff
@@ -1,43 +0,0 @@
-#!/bin/bash
-
-# The website is built using MkDocs with the Material theme.
-# https://squidfunk.github.io/mkdocs-material/
-# It requires Python to run.
-# Install the packages with the following command:
-# pip install mkdocs mkdocs-material
-
-# Generate the API docs
-./gradlew dokka
-
-# Dokka filenames like `-http-url/index.md` don't work well with MkDocs <title> tags.
```

Comment:
I haven't found any symptoms of the problem described here, was it fixed in MkDocs at some point? The rest of the commands in this file aren't relevant anymore.